include/swift/SwiftRemoteMirror/Platform.h

```diff
@@ -41,6 +41,13 @@ extern "C" {
 # endif
 #endif
 
+#if defined(__clang__)
+#define SWIFT_REMOTE_MIRROR_DEPRECATED(MSG, FIX)                               \
+  __attribute__((deprecated(MSG, FIX)))
```

Comment:
Would be nice to use the reserved spelling: `__attribute__((__deprecated__(MSG, FIX)))`

Reply:
Done.